tests/queryset_pickle/tests.py

```diff
@@ -115,6 +115,38 @@ def test_pickle_prefetch_related_idempotence(self):
         groups = pickle.loads(pickle.dumps(groups))
         self.assertSequenceEqual(groups, [g])
 
+    def test_pickle_prefetch_queryset_not_evaluated(self):
```

Comment:
The other tests were added as I changed the implementation in the previous commits, as per @charettes feedback ^